tests/infrared/17.1/stf-connectors.yaml.template

```diff
@@ -0,0 +1,68 @@
+---
+tripleo_heat_templates:
+    []
+
+custom_templates:
+    # don't load collectd-write-qdr.yaml when using multi-cloud and instead load collectd service directly
+    resource_registry:
+        OS::TripleO::Services::Collectd: /usr/share/openstack-tripleo-heat-templates/deployment/metrics/collectd-container-puppet.yaml
+
+    # set parameter defaults to match stable-1.3 documentation
+    parameter_defaults:
+        MetricsQdrConnectors:
+            - host: <<AMQP_HOST>>
+              port: <<AMQP_PORT>>
+              role: edge
+              verifyHostname: false
+              sslProfile: sslProfile
+
+        MetricsQdrSSLProfiles:
+            - name: sslProfile
+              caCertFileContent: |
+                <<CA_CERT_FILE_CONTENT>>
+
+        CeilometerQdrEventsConfig:
+            driver: amqp
+            topic: <<CLOUD_NAME>>-event
+
+        CeilometerQdrMetricsConfig:
+            driver: amqp
+            topic: <<CLOUD_NAME>>-metering
+
+        CollectdAmqpInstances:
+            <<CLOUD_NAME>>-notify:
+                format: JSON
+                notify: true
+                presettle: false
+            <<CLOUD_NAME>>-telemetry:
+                format: JSON
+                presettle: false
+
+        CollectdSensubilityResultsChannel: sensubility/<<CLOUD_NAME>>-telemetry
+
+        # --- below here, extended configuration for environment beyond what is documented in stable-1.3
+        CollectdSensubilityLogLevel: DEBUG
+        CephStorageExtraConfig:
+            tripleo::profile::base::metrics::collectd::amqp_host: "%{hiera('storage')}"
+            tripleo::profile::base::metrics::qdr::listener_addr: "%{hiera('storage')}"
+
+        ExtraConfig:
+            ExtraHostsEntries:
+                - "<<EXTRA_HOSTS_ENTRIES>>"
+
+            collectd::plugin::ceph::daemons:
```

Comment:
Those are a lot of OSDs

Reply:
Yea this is all pretty much copy/pasta from previous generations of scripts. You can see these files in the various OSP matching version directories (see 170, 16.2, 13, etc)